lenskit/algorithms/mf_common.py

```diff
@@ -141,12 +143,12 @@ def score(self, user, items, raw=False):
             numpy.ndarray: the scores for the items.
         """
 
-        rv = super().score(user, items)
+        rv = super().score(user, items, user_feature)
 
         if not raw:
             # add bias back in
             rv = rv + self.global_bias_
-            if self.user_bias_ is not None:
+            if self.user_bias_ is not None and user is not None:
```

Comment:
So: for a new user, this means it won't add _this_ user's bias back in.

The clean solution to this problem is to do a bit of updating on the `Bias` class, and use it instead of reproducing bias-oriented logic in the matrix factorizers. Issue #174 discusses such changes.